src/main/java/christmas/domain/order/OrderPrice.java

```diff
@@ -0,0 +1,22 @@
+package christmas.domain.order;
+
+import christmas.constants.Menu;
+import java.util.Map;
+
+public class OrderPrice {
+    private static final int NO_ORDER = 0;
+
+    public int calculateTotalOrderPrice(Map<String, Integer> orderTable) {
+        int totalOrderPrice = NO_ORDER;
+        for (String menuName : orderTable.keySet()) {
+            int price = Menu.getPriceByName(menuName);
```

Comment:
위에서 Menu 클래스의 getPriceByName 함수의 로직을 보고, 이름을 가지고 메뉴를 찾는 경우가 어떤거지? 어디에 사용되지? 했었는데 여기서 사용되었군요!

제 생각에 orderTable의 메뉴 이름 값을 String으로 지정하면서 성능이 조금 떨어지게 되는 것 같아요! Menu는 enum이라 서로 상충되기 때문이죠 🤔
그래서! 혹시, orderTable의 메뉴 이름 부분의 타입을 Menu 클래스로 지정해준다면 어떨까요?
```suggestion
            int price = menuName.getPrice();
```
처럼 간단해질 것이라고 생각하네요
어떻게 생각하시나요?! orderTable의 메뉴 이름을 String 타입으로 지정하신 의도가 있으신지 궁금합니다!!

Reply:
클래스를 타입으로서 활용하는 데 미숙해서 그런 것 같아요... '클래스는 값이 아니지 않나? 결국 String, int와 같은 형태로 데이터를 가져와야 하는데...어느 지점에서 가져와야 하지?' 등과 같은 생각이 드는 것 같네요.😢